src/caret_analyze/architecture/struct/node.py

```diff
@@ -225,13 +201,33 @@ def update_node_path(self, paths: list[NodePathStruct]) -> None:
     #     # TODO: Refactoring module dependency
     #     pass
 
-    def insert_publisher_callback(self, publish_topic_name: str, callback_name: str) -> None:
+    def insert_publisher_callback(self,
+                                  publish_topic_name: str,
+                                  callback_name: str,
+                                  publisher_construction_order: int) -> None:
         callback: CallbackStruct = \
             Util.find_one(lambda x: x.callback_name == callback_name, self.callbacks)
-        callback.insert_publisher(publish_topic_name)
+        try:
+            pub_info = PublishTopicInfoValue(publish_topic_name, publisher_construction_order)
+            callback.insert_publisher(pub_info)
+        except MultipleItemFoundError as e:
+            msg = f'{e}'
+            raise MultipleItemFoundError(msg)
```

Comment:
I suggest removing this `try ... exept` because `MultipleItemFoundError` will not be raised by these lines.

```suggestion
        pub_info = PublishTopicInfoValue(publish_topic_name, publisher_construction_order)
        callback.insert_publisher(pub_info)
```

Reply:
Sorry, my last comment is wrong. `callback.insert_publisher()` will raise `MultipleItemFoundError`. Still, this change looks good to me because there is no information to be added to `MultipleItemFoundError` here.

I confirmed that this change is already included in this PR.